cmd/runtime_options.go

```diff
@@ -60,9 +60,9 @@ func runtimeOptionFlagSet(includeSysEnv bool) *pflag.FlagSet {
 	flags.Bool("include-system-env-vars", includeSysEnv, "pass the real system environment variables to the runtime")
 	flags.String("compatibility-mode", "extended",
 		`JavaScript compiler compatibility mode, "extended" or "base"
-base: pure Golang JS VM supporting ES5.1+
-extended: base + Babel with ES2015 preset + core.js v2,
-          slower and memory consuming but with greater JS support
+base: pure goja - Golang JS VM supporting ES5.1+
+extended: base + Babel with parts of ES2015 preset
```

Comment:
I wonder if we want to just list classes and import/export here?

Reply:
It would be too much information for this help output, so no, but we could mention it in the docs.